Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Slate/HandInteractionPanZoom.cs

```diff
@@ -722,15 +703,13 @@ private void StartTouch(uint sourceId)
         {
             UpdateTouchUVOffset(sourceId);
             FirePanStarted(sourceId);
-            PanStarted?.Invoke();
         }
         private void EndTouch(uint sourceId)
         {
             if (handDataMap.ContainsKey(sourceId) == true)
             {
                 handDataMap.Remove(sourceId);
                 FirePanEnded(0);
```

Comment:
Rename "Fire" to "Raise", i believe that's what we use for events throughout mrtk.